app/controllers/receipts_controller.rb

```diff
@@ -0,0 +1,69 @@
+class ReceiptsController < ApplicationController
+  include CartsHelper
+  before_action :store_location, :logged_in_user
+  before_action :load_cart, :load_room, only: :create
+
+  def index
+    @receipts = Receipt.of_current_user current_user
+  end
+
+  def show
+    @receipt = Receipt.find_by id: params[:id]
+    return if current_user.id == @receipt.user_id
```

Comment:
```suggestion
    @receipt = @current_user.receipts.find_by id: params[:id]
    return if @receipt
```

@receipt mà nill gọi user_id có sống dc ko